apps/src/templates/progress/ProgressLevelSet.jsx

```diff
@@ -85,6 +85,26 @@ const ProgressLevelSet = React.createClass({
     levels: PropTypes.arrayOf(levelType).isRequired
   },
 
+  getIcon() {
+    const { levels } = this.props;
+    const level = levels[0];
+
+    // TODO - Once we know what locked levels and peer reviews are going to look
+    // like in the redesign, we'll need add logic for those here.
+
+    if (level.icon) {
+      // Eventually I'd like to have dashboard return an icon type. For now, I'm just
+      // going to treat the css class it sends as a type, and map it to an icon name.
```

Comment:
Agreed - `fa-list-ol` is a display concern, and not something dashboard should have any knowledge of.  When are we going to come up with a nice way to share enums between rails and javascript? :grin: